lib/cn/kvset.c

```diff
@@ -2288,7 +2288,7 @@ kblk_start_read(struct kvset_iterator *iter, struct kblk_reader *kr, enum read_t
     assert(iter->workq);
 
     if (kr->kr_nodex == kr->kr_nodec) {
-        struct wbt_desc *wbt;
+        struct wbt_desc *wbt = NULL;
```

Comment:
Fix -Og complaints.  There are others we need to fix in spill.c, but I didn't address them in this patch.